python/pyspark/sql/types.py

```diff
@@ -189,7 +189,7 @@ def toInternal(self, dt):
         if dt is not None:
             seconds = (calendar.timegm(dt.utctimetuple()) if dt.tzinfo
```

Comment:
I will, yes. Trying to find where the appropriate location is in the Scala code.